src/operator/tensor/elemwise_unary_op.h

```diff
@@ -562,6 +570,50 @@ struct ReshapeLikeParam : public dmlc::Parameter<ReshapeLikeParam> {
     })                                                              \
   .add_argument("data", "NDArray-or-Symbol", "The input array.")
 
+/*! \bried MKL Unary compute. 
+ *  With this macro means mxnet compile with MKL to accelerate math function with mkl. 
+ *  Will Register FCompute with UnaryOp::MKL_Compute() to compelet the math function. 
+ */
+#define MXNET_MKL_OPERATOR_REGISTER_UNARY(__name$)                  \
```

Comment:
How is this different from `MXNET_OPERATOR_REGISTER_UNARY`?

Reply:
The implementation of these two macros  are same, duplicates code removed.